### DISCUSSION TITLE
what regexp syntax does Ripgrep use?

### DISCUSSION THREAD

**Discussion by achenet** [BurntSushi/ripgrep#2870] [Q&A] [answered]
what regexp syntax does Ripgrep use?
I can't find the info when reading the guide?

**Comment by BurntSushi** [comment, marked as answer]
It's [linked](https://docs.rs/regex/1.10.6/regex/#syntax) directly [from the README](https://github.com/BurntSushi/ripgrep?tab=readme-ov-file#documentation-quick-links).
